CMakeLists.txt

```diff
@@ -217,14 +218,61 @@ set(CPACK_PACKAGE_VERSION_PATCH ${VERSION_PATCH})
 set(CPACK_PACKAGE_VENDOR "celeron55")
 set(CPACK_PACKAGE_CONTACT "Perttu Ahola <celeron55@gmail.com>")
 
+include(CPackComponent)
+
+cpack_add_component(SUBGAME_MINETEST_GAME
+	DISPLAY_NAME "Minetest Game"
+	DESCRIPTION "The official subgame for the Minetest engine, that can easily extended by mods."
+	GROUP "Subgames"
+)
+
+cpack_add_component(SUBGAME_MINIMAL
+	DISPLAY_NAME "Minimal development test"
+	DESCRIPTION "A minimal subgame helping to develop the engine."
+	DISABLED #DISABLED does not mean it is disabled, it means its not selected on default!
+	GROUP "Subgames"
+)
+
+cpack_add_component_group(Subgames
+	DESCRIPTION "Games for the Minetest engine."
+)
+
 if(WIN32)
-	if(CMAKE_SIZEOF_VOID_P EQUAL 8)
-		set(CPACK_PACKAGE_FILE_NAME "${PROJECT_NAME}-${VERSION_STRING}-win64")
+	# Include all dynamically linked runtime libaries
+	include(InstallRequiredSystemLibraries)
+
+	if(RUN_IN_PLACE)
```

Comment:
> So what happens if I compile using MinGW on Linux (-> no WiX toolset) with RUN_IN_PLACE=0 and run make package?

a) nothing
b) it prints an error message
c) it prints a warning and generates a zip package anyway
d) it generates a zip package without any other message
e) it downloads and installs the wix toolset and generates a msi package

Well I don't know. 😆  How did you generate your 7zip packages you uploaded [here](https://minetest.kitsunemimi.pw/builds/)? You could probably do the same only as zip format.

> Why? It would use the exact same runtime paths as if it was installed using the installer.

The problem is to tell the kiddies that "zip package 1" finds its mods and conf in a different place then "zip package 2" 🤔